test/core/tsi/ssl_transport_security_test.cc

```diff
@@ -727,9 +727,18 @@ void test_tsi_ssl_server_handshaker_factory_refcounting() {
   cert_pair.private_key =
       load_file(SSL_TSI_TEST_CREDENTIALS_DIR, "server0.key");
 
-  GPR_ASSERT(tsi_create_ssl_server_handshaker_factory(
-                 &cert_pair, 1, cert_chain, 0, nullptr, nullptr, 0,
-                 &server_handshaker_factory) == TSI_OK);
+  tsi_ssl_server_handshaker_options options;
+  memset(&options, 0, sizeof(options));
```

Comment:
similarly, I this this memset would be unnecessary if you add 0 initialization to the definition of this struct.

Reply:
ssh_transport_security.h is a C header. We have not c++'fying it yet.
I am worrying changing to c++ struct may break the consistency.

Reply:
Everything that's in `src/core/...` is compiled as C++. Setting the initial values would be just fine. I have done this a few times now :-) 

But, I don't feel strongly. So, I'll approve and submit if you prefer it this way.

Reply:
SG. Updated. PTAL